eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java

```diff
@@ -162,8 +173,15 @@ public <T extends AbstractMojo> FakeMaven execute(final Class<T> mojo) throws IO
                 tojo.set(entry.getKey().key(), entry.getValue());
             }
         }
+        for (final Tojo tojo : this.external().select(all -> true)) {
```

Comment:
@maxonfjvipon Looks like a code duplication. Can we create a common method for the both tojos, or just add a puzzle?